### PR TITLE
Enable CircleCI builds

### PR DESCRIPTION
**Pull request by berkus**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,226 @@
+version: 2.1
+
+aliases:
+  - &workspace
+    ~/twilio-boost-build
+  - &build_output
```
Comment (Gray-Wind) [inline .circleci/config.yml:6]: This is never used

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+      boost-libs:
+        description: "List of boost libraries to build"
+        type: string
+        default: "atomic container context coroutine date_time exception iostreams program_options random regex serialization system test thread"
+  - pre-steps-parameter: &pre-steps-parameter
+      pre-steps:
+        description: "Steps that will be executed before build starts"
+        type: steps
+        default: []
+  - post-steps-parameter: &post-steps-parameter
```
Comment (Gray-Wind) [inline .circleci/config.yml:27]: Will it be used?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+      post-steps:
+        description: "Steps that will be executed after build ends"
+        type: steps
+        default: []
+  - build-steps-parameter: &build-steps-parameter
```
Comment (Gray-Wind) [inline .circleci/config.yml:32]: Is it used

**Review by Pavel-Cherentsov — APPROVED** [review]

File: boost.sh [hunk 4]
```diff
@@ -1069,33 +1092,23 @@ deployToBintray()
...
+    SETTINGS_FILE="$CURRENT_DIR/bintray-settings.xml"
 
     SETTINGS_FILE="-s $SETTINGS_FILE"
 
     deployToNexus
 }
 
+markBintrayDeployed()
+{
+    echo "All is published, hurray!"
```
Comment (Gray-Wind) [inline boost.sh:1105]: Would it be fixed in this PR?

Comment (berkus) [reply boost.sh:1105]: Nope

**Pull request merged**